package.json

```diff
@@ -172,4 +172,4 @@
     "electron-packager": "^14.2.1",
     "electron-winstaller": "4.0.0"
   }
-}
+}
```

Comment:
Let's keep the end line here to not have it changed back in another PR

```suggestion
}

```

Reply:
I have no preference - I leave it in whatever state it's in after `lint:fix` runs.

Reply:
And it looks like `lint:fix` took care of it in the second commit. :smile:

Reply:
Ah cool, nevermind then 😄